lib/inspec/reporters.rb

```diff
@@ -0,0 +1,32 @@
+require 'inspec/reporters/base'
+require 'inspec/reporters/cli'
+require 'inspec/reporters/json'
+require 'inspec/reporters/json_min'
+require 'inspec/reporters/junit'
+
+module Inspec::Reporters
+  def self.render(config)
+    case config[:name]
+    when 'cli'
+      reporter = Inspec::Reporters::CLI.new(config)
+    when 'json'
+      reporter = Inspec::Reporters::Json.new(config)
+    when 'json-min'
+      reporter = Inspec::Reporters::JsonMin.new(config)
+    when 'junit'
+      reporter = Inspec::Reporters::Junit.new(config)
+    else
```

Comment:
Just to be sure I understand the implementation correct, the `html` & `documentation`  reporter just works by accident?
@adamleff What do you think? Should this be `rspec-documentation`, `rspec-html`?

Reply:
Not by accident... on purpose :) We handle that in the runner because that's where we need to invoke and register the RSpec-provided formatters. We have lots of users in the community that use the doc and html formatters, especially in CI pipelines or Test Kitchen.  This class is just setting up our new Reporter stuff that we own.

I'd vote against changing the names for two reasons:

 - it would be a breaking change, and lots of our TK community use the documentation formatter
 - it further calls attention to the fact that RSpec is used, and I'd love to be able to hide RSpec details from our users if at all possible.

That said, if further separation is required in the future, I'd totally support it. It would just need to be in a major version bump 3.0 or later at this point.

Reply:
That makes sense @adamleff